TFNetwork.py

```diff
@@ -1411,31 +1411,45 @@ def get_norm_factor(self):
     self._prepare()
     return self._norm_factor
 
-  def _normalized_loss_value_per_seq(self, value):
+  def _normalized_value_per_seq(self, value, per_pos=False):
     """
-    :param tf.Tensor|None loss:
+    :param tf.Tensor|None value: (batch*time,) or (time*batch,)
+    :param bool per_pos: if true return tf.Tensor(B,T) else tf.Tensor(B)
```

Comment:
Keep the comments consistent. Write `... (batch,time) else (batch,)`.
Or actually this should rather be written for `:return:`.